core/server/src/main/java/alluxio/master/file/meta/LockedInodePath.java

```diff
@@ -71,6 +71,14 @@ public synchronized Inode getInode() throws FileDoesNotExistException {
   }
 
   /**
+   * @return the target inode which is guaranteed to exist
+   */
+  public synchronized Inode getInodeNoException() {
```

Comment:
Done. I think it makes code less readable by catching the exception that is never going to happen.